docs/src/PolyhedralGeometry/Polyhedra/pg_polyhedra_polymake.md

```diff
@@ -0,0 +1,55 @@
+```@meta
+CurrentModule = Oscar
+```
+
+```@setup oscar
+using Oscar
+```
+
+```@contents
+Pages = ["pg_polyhedra_polymake.md"]
+```
+
+
+# `Polyhedron` and `polymake`'s `Polytope`
+
+Many polyhedral computations are done through `polymake`.
+This is visible in the structure `Polyhedron` via a pointer `pm_polytope` to the corresponding `polymake` object.
+This allows to apply all functionality of `polymake` by calling a suitable `Polymake.jl` function on this pointer.
+
+To allow both `Oscar`'s and `polymake`'s functionality to be applicable to a polyhedron object, it can be converted back and forth:
+
+```@docs
+Polyhedron(::Polymake.BigObject)
+pm_polytope(::Polyhedron)
+```
+
+The following shows all the data "known to" a `Polyhedron`.
+
+```@repl oscar
+C = cube(3)
+C.pm_polytope
+```
+
+`polymake` allows for an interactive visualization of 3-dimensional polytopes in the browser: `Polymake.visual(C.pm_polytope)`.
+
+There are several design differences between `polymake` and `Oscar`.
```

Comment:
... And... ? Are you going to tell us? Or is the "warning" coming next part of this message? If so, it should be made clear, e.g.:
```suggestion
There are several design differences between `polymake` and `Oscar`. The most important are the following:
```
But actually, it might be better to either ditch this sentence, or move it into that "warning"...